.travis.yml

```diff
@@ -3,17 +3,17 @@ sudo: required
 env:
     matrix:
         # openssl tests
-        - COMP=openssl OS_TYPE=centos OS_VERSION=5
+        #- COMP=openssl OS_TYPE=centos OS_VERSION=5
         - COMP=openssl OS_TYPE=centos OS_VERSION=6
         - COMP=openssl OS_TYPE=centos OS_VERSION=7
         - COMP=openssl OS_TYPE=fedora OS_VERSION=24
```

Comment:
in general, for Fedora, we care only about `latest`

I think it's a bit too early to say and we have too little resources to run on multiple Fedora versions, the probability of regressions is highest in newest versions anyway